tree/ntupleutil/v7/src/RNTupleExporter.cxx

```diff
@@ -162,7 +162,7 @@ RNTupleExporter::RPagesResult RNTupleExporter::ExportPages(RPageSource &source,
             const std::uint64_t pageBufSize = pageInfo.fLocator.GetNBytesOnStorage() + maybeChecksumSize;
             std::ostringstream ss{options.fOutputPath, std::ios_base::ate};
             ss << "/cluster_" << clusterDesc.GetId() << "_" << colInfo.fQualifiedName << "_page_" << pageIdx
-               << "_elems_" << pageInfo.fNElements << "_comp_" << colRange.fCompressionSettings << ".page";
+               << "_elems_" << pageInfo.fNElements << "_comp_" << colRange.fCompressionSettings.value() << ".page";
```

Comment:
Shouldn't this properly handle the case where compression is unknown, rather than throwing?

Reply:
At this point, we know that the column is neither suppressed nor in the deferred range. I added an assert and changed the access to the unchecked one.